packages/plugins/other/visitor-plugin-common/src/selection-set-to-object.ts

```diff
@@ -383,7 +383,10 @@ export class SelectionSetToObject<Config extends ParsedDocumentsConfig = ParsedD
       ...(typeInfoField ? this._processor.transformTypenameField(typeInfoField.type, typeInfoField.name) : []),
       ...this._processor.transformPrimitiveFields(
         parentSchemaType,
-        Array.from(primitiveFields.values()).map(field => field.name.value)
+        Array.from(primitiveFields.values()).map(field => ({
+          makeNullable: this.resolveDirectives(field.directives),
```

Comment:
Maybe we can move `resolveDirectives` method to common library or plugin-helpers or something? 
And maybe we can change the name to `hasConditionalDirectives`?